<?xml version="1.0"?>
<launch>
  <node pkg="smb_navigation_scripts" type="filter_pointcloud.py" name="filter_pointcloud" output="screen" required="true" >
    <param name="clear_radius" value="0.25"/>
    
    <!-- Remapping -->
    <remap from="rslidar/points"          to="rslidar/points"/>
    <remap from="rslidar/points_filtered" to="rslidar/points_filtered"/>
  </node>
</launch>
